apps/admin-x-framework/src/providers/FrameworkProvider.tsx

```diff
@@ -4,6 +4,18 @@ import {ReactNode, createContext, useContext} from 'react';
 import queryClient from '../utils/queryClient';
 import {ExternalLink} from './RoutingProvider';
 
+// Stats-specific configuration
```

Comment:
I added StatsConfig to the admin-x-framework to be able to use it across multiple React apps. However, I want to merge the current `stats` and `posts` app so this is a temporary state